aidb/inference/examples/huggingface_inference_service.py

```diff
@@ -15,6 +15,7 @@ def __init__(
       default_args: Dict[str, Union[str, int]]=None,
       columns_to_input_keys: Dict[str, Union[str, tuple]]=None,
       response_keys_to_columns: Dict[Union[str, tuple], str]=None,
+      gap_between_requests: float=0.0,
```

Comment:
Can you change this to a rate limit instead and perform the calculation for gap between requests internally?

Reply:
I'll do that